webapp/channels/src/utils/performance_telemetry/reporter.ts

```diff
@@ -341,6 +359,34 @@ export default class PerformanceReporter {
 
         return navigator.sendBeacon(url, data);
     }
+
+    protected collectDesktopAppMetrics(metricsMap: Map<string, {cpu?: number; memory?: number}>) {
```

Comment:
Is there any risk of the process name leaking anything potentially identifiable like if someone whitelabels their desktop app? I'm trying to understand if it's just generic names like "renderer" or if it's the full "Mattermost Helper (Renderer)" name

Reply:
It's all generic internal names from Electron, it would only leak what we name the processes that are attached to it. So any of those names that are attached to the `PerformanceMonitor` in the Desktop App should just be a constant.